src/main/resources/static/openapi/SupportAdditionalNeedsAPI.yml

```diff
@@ -1473,6 +1492,25 @@ components:
           description: |
             An ISO-8601 date representing date that the plan creation is due.
           example: '2023-11-19'
+        planCompletedDate:
+          type: string
+          format: date
+          description: |
+            An ISO-8601 date representing date that the plan creation was created.
+          example: '2023-11-19'
+        planCompletedBy:
+          type: string
+          description: |
```

Comment:
How about

> If the status of this Plan Creation Schedule is COMPLETED, and the person who met with the prisoner to create their Education Support Plan was not the same person who keyed it into the SAN service, this field will be that person's name.
This field will only have a value when the status of the Plan Creation Schedule is COMPLETED, and the person who met with the prisoner to create their Education Support Plan was not the same person who keyed it into the SAN service. If the Plan Creation Schedule is COMPLETED and this field is null, consumers of this API can assume the person who created the Education Support Plan is the person who keyed it in. See field `planKeyedInBy`

Reply:
```
example: Joe Bloggs
```